### PR TITLE
DOCS Clarified Core vs. Supported Modules, added "Triage" role

### PR DESCRIPTION
**Pull request by chillu**
[ci-skip]

### CODE REVIEW

**Review by sminnee — APPROVED** [review]
Seems fine.

**Comment by chillu** [comment]
@silverstripe/core-team I'm suggesting a new "Triage" role for core, with more limited permissions than "Contributing Committer" - opening up a few more pathways to help out without getting the keys to the castle right away. The immediate use case is @adrhumphreys who wants to help out on `silverstripe/assets`, but I reckon this can be useful in other contexts as well. Any issues with this? You can read up about Github roles in https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization

File: docs/en/05_Contributing/09_Core_committers.md [hunk 1]
```diff
@@ -55,3 +70,23 @@ often working alongside Core Committers. They are guided by additional rules:
...
+
+### Triage
+
+Triage of issues and pull request is an important activity: Reviewing issues, adding labels,
+closing stale issues, etc. This does not require write access to the repository as a "Contributing Committer".
+This is a great way for active community members to help out, and start a path towards becoming a Core Committer.
+
+Triage roles may be granted by any Core Committer,
+who should give other Core Committers an opportunity to weigh in on the decision.
+## Supported Modules
```
Comment (michalkleiner) [inline docs/en/05_Contributing/09_Core_committers.md:85]: Blank line above?

**Comment by ScopeyNZ** [comment]
LGTM. I'm pretty sure we've had a triage role in the past that the UX team at SilverStripe has used to triage issues. I guess we never wrote that in our docs though?

There is already a triage team with two members.

**Review by dnsl48 — APPROVED** [review]
lgtm

File: docs/en/05_Contributing/09_Core_committers.md (hunk 1 above)
Comment (chillu) [reply docs/en/05_Contributing/09_Core_committers.md:85]: Hah thanks Michal, started using VSCode and it has this thing where it shows the `git blame` in a "fake line" above, which makes it easy to miss when there's no actual blank line. Fixed now.

**Review by dhensby — APPROVED** [review]

**Pull request merged**